static/app/components/nav/utils.tsx

```diff
@@ -1,31 +1,49 @@
-import type {ComponentProps} from 'react';
 import type {LocationDescriptor} from 'history';
 
-import type Feature from 'sentry/components/acl/feature';
+import type {FeatureProps} from 'sentry/components/acl/feature';
+import type {MenuItemProps} from 'sentry/components/dropdownMenu';
 import {isItemActive} from 'sentry/components/sidebar/sidebarItem';
 import {SIDEBAR_NAVIGATION_SOURCE} from 'sentry/components/sidebar/utils';
 import normalizeUrl from 'sentry/utils/url/normalizeUrl';
 import type {useLocation} from 'sentry/utils/useLocation';
 
-/** NavItem is the base class for both SidebarItem and SubmenuItem  */
+/**
+ * NavItem is the base class for both SidebarItem and SubmenuItem
+ */
```

Comment:
nit: would be nice to see these in a different PR since (these are changes!) this clutters up this PR a bit